middleware/middleware-1.3.36.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: /release-notes/middleware/1.3.36
+title: Version 1.3.36
+---
+
+# fiskaltrust.Middleware 1.3.36 (Germany)
+_July 21, 2022_
+
+In this version of the Middleware, we've added Database indices for frequently accessed fields and localization to our Android App.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Stability improvement: Added necessary indices to MW databases to improve query performance
+As we now have plenty of operational experience, we should introduced indices to the databases so that frequent queries (e.g. while doing /sign or /journal calls) are optimized. The Amount of RW operations on the HDD should be significantly reduced.
```

Comment:
```suggestion
With the operational experience we have collected since the go-live in Germany, we have added new indices to the Middleware database so that frequent queries (e.g. while doing /sign or /journal calls) are optimized. The Amount of RW operations on the HDD should be significantly reduced, as well as the execution time.
```